### ISSUE TITLE
[a11y][FxA] Review a11y across the app, including Send Tab

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2773]
### Vision statement / What / Requirements
It's been a while since we've checked our ability to be used with a screen reader: we should ensure it's working. In particular, we just finished the Send Tab functionality so we ensure that is working too.

### Impact
Screen reader users may be unable to use these features without this testing.

### Acceptance criteria
- Every screen and user flow in the app has been tested with the VoiceView screen reader
  - In particular, all Send Tab functionality has been tested with the VoiceView screen reader
- Bugs are filed for issues found

**Comment by mcomella** [comment]
@athomasmoz I wonder if we should include a11y in the original acceptance criteria, like we include telemetry. I wonder if it'd be more efficient at addressing these issues or make us less forgetful to do this kind of testing.

**Comment by mcomella** [comment]
Repurposing this bug to deal with a11y across the entire app.

**Comment by athomasmoz** [comment]
Can we do an initial pass on this? May look for expertise from A11y team later, but we could do a pass to start.

**Comment by mcomella** [comment]
I have completed a review of the standard portions of the app and filed bugs. I still need to:
- Test review mode on FxA login (which is likely non-standard for users)
- Test out non-standard FxA flows
  - Create account
  - Invalid email, password
  - Enter an email that doesn't have an account
  - Use "Forgot password" option

**Comment by mcomella** [comment]
I tested review mode and updated the already filed bugs accordingly: it's a better experience besides the fact that we can't open text fields without disabling review mode: #2935.

The remaining work here has diminishing returns, I think, so I filed a follow-up to track instead: https://github.com/mozilla-mobile/firefox-tv/issues/2936
